browser-api/src/content.js

```diff
@@ -50,6 +50,7 @@ function iframeFinder() {
 
             //Create Warning Div
             let warning = document.createElement("div");
+            warning.classList.add("warning-" + rid);
```

Comment:
Went on with `warning-XXXX` rather than just `XXXX` so that we don't unwillingly target any minified css class name from some UI framework element that actually existed on the website that happened to be exactly `XXXX`